banana-in-box should output '@angular-eslint/template-parser' as the parser

<!--
👋 Hi, thanks for filing an issue on tslint-to-eslint-config! 💖
Please fill out all fields below to ensure your issue is addressed.

If your issue doesn't provide enough info to fully explain or reproduce your bug, it will be closed. 😦
-->

### 🐛 Bug Report

-   `tslint-to-eslint-config` version: current
-   ESLint version: 7.2.0
-   Node version: 14.15.5

#### Actual Behavior
1st problem:

```
2 rules are not known by tslint-to-eslint-config to have ESLint equivalents. ❓
  The "@typescript-eslint/tslint/config" section of ./.eslintrc.js configures eslint-plugin-tslint to run them in TSLint within ESLint.
  * tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "import-spacing".
  * tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "whitespace".
```

2nd problem:
 All is well! ✅
... then, while first eslint

```
Oops! Something went wrong! :(
Error: Error while loading rule '@angular-eslint/template/banana-in-box': You have used a rule which requires '@angular-eslint/template-parser' to be used as the 'parser' in your ESLint config.
```

3rd problem
After changing the parser value:

```
Oops! Something went wrong! :(
Error: Failed to load parser '@angular-eslint/template-parser' declared in '.eslintrc.js': Cannot find module '@angular-eslint/template-parser'
```

4th problem
After installing the package

```
Oops! Something went wrong! :(
Error: Error while loading rule '@typescript-eslint/dot-notation': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.
Looking into .eslintrc.js:
    "parserOptions": {
        "project": "tsconfig.json",
        "sourceType": "module"
    },
```
Huh?

#### Expected Behavior

A smooth transition for a standard angular project, importing all rules, especially the whitespace-rule, setting the correct config, and not leaving me behind clueless with a misleading error message.

#### Reproduction

https://github.com/rongem/cmdb/tree/master/ng-frontend

--- a/README.md
+++ b/README.md
@@ -21,6 +21,8 @@ Consider taking a peek at the relevant documentation: 🤔
 
 ## Usage
 
+> ⚠ Coming from Angular? Use **[angular-eslint](https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint)** instead to coordinate around this tool and others.
+
 ```shell
 npx tslint-to-eslint-config
 ```
@@ -161,7 +163,6 @@ _Default: `tsconfig.json`_
 
 Path to a TypeScript configuration file to read TypeScript compiler options from.
 This will help inform the generated ESLint configuration file's [env](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) settings.
-
 
 ## Node API
 

